src/app/layout/header/header.component.html

```diff
@@ -3,20 +3,20 @@
   <nav class="navbar topnavbar" role="navigation">
     <!-- START navbar header-->
     <div class="navbar-header" id="navbarHeader">
-      <a routerLink="/dashboard">
+      <a routerLink="/dashboard" data-qa-id="logo">
```

Comment:
is `data-qa-id` a standard marker for `frontend` testing? A quick google found that maybe sometimes just `data-qa` is used? https://dev.to/chriszie/data-qa-attribute-a-better-way-to-select-elements-for-ui-test-automation-48lm
What is your take on this?

Reply:
Habit :D With colleague we decided to do something like this and i used to it.
Cypress is suggesting to use `data-cy="..."`, it depends on the team. I can always change this one to be `data-qa="..."` but i do not see a point.